pkg/ibmvpc/parser.go

```diff
@@ -1029,6 +1080,9 @@ func getIKSnodesConfig(res vpcmodel.MultipleVPCConfigs,
 					SubnetResource: subnet,
 				},
 			}
+			res[vpcUID].UIDToResource[nodeObject.ResourceUID] = nodeObject
+			addIKSNodeAsSGTarget(sg, iksCluster)
```

Comment:
why is `addIKSNodeAsSGTarget` called foe each worker node, if it iterates all worker nodes in the cluster and adds them as SG targets? 
Shouldn't it be called outside the loop?

Reply:
because we associate each iks worker node to it's SG not each iks cluster

Reply:
so why on each worker node you call `addIKSNodeAsSGTarget` with `iksCluster`, and then within this function you iterate all cluster nodes?

Reply:
right should be after the sec loop